### ISSUE TITLE
Allow the pass-through of the "custom" field for PayPal

### ISSUE THREAD

**Issue by davidmcl** [recurly/recurly-js#215]
Looking at the recurly.js docs, the `description` field is the only one that is listed, so I'm guessing that's all that is available at this point.

I would like to request that the `custom` field is made available as an optional parameter.
It would be quite helpful for those of us who have to support a legacy implementaion of PayPal until that zeros-out in addition to transitioning to the Recurly system PayPal support.  

The `custom` field description in PayPal is;

```
Custom value as passed by you, the merchant. 
These are pass-through variables that are never presented to your customer 
Length: 255 characters
```

Basically, I need to pass info into the trans and get it back in IPN and differentiate between Recurly and legacy implementaions and I suspect that the `custom` field is pretty much the only way to accomplish that consistently between the two implementations. 

Thanks,
David
